docs/config.rst

```diff
@@ -0,0 +1,778 @@
+
+
+
+Override Default Values
+^^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+    These overrides pertain only to scratch orgs.
+
+You can override these values for your org.
+
+* ``days`` (integer): Number of days for the scratch org to persist.
+* ``namespaced`` (boolean): Is the scratch org a namespaced or not.
+* ``config_file`` (string): Path to the org definition file to use when building the scratch org.
+
+.. code-block:: yaml
+
+    orgs:
+        scratch:
+            <org_name>:
+                <key>: <value>
+
+Replace all objects with ``<>`` with the desired values.
+
+For example, override the default number of days from 7 to 15 in the ``dev`` org.
+
+.. code-block:: yaml
+
+    orgs:
+        dev:
+            days: 15
+
+
+
+Configuration Scopes
+--------------------
+
+CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that enable configuration at several distinct scopes. All of these files have the same name, ``cumulusci.yml``, but live in different locations in the file system.
+
+You can configure files at these scope levels: *Project*, *Local Project* and *Global*. Configurations have an order of override precedence (from highest to lowest):
+
+#. Project
+#. Local Project
+#. Global
+
+One override only cascades over another when two configurations set a value for the same element on a task or flow.
+
+Take for example, task ``T`` which takes two options, ``opt1`` and ``opt2``.
+
+You can specify a default value for ``opt1`` in your project ``cumulusci.yml`` file and a default value for ``opt2`` in your global ``cumulusci.yml`` file,
+and you'll see the expected result: both values are available in the project.
+(The default of ``opt1`` is not exposed to other projects.)
+
+If you change your project ``cumulusci.yml`` file to also specify a default value for ``opt2``, this new default ``opt2`` value takes precedence over the default ``opt2`` value specified in your global ``cumulusci.yml`` file.
+
+
+Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``.../path/to/project/cumulusci.yml``
+
+**Windows:** ``...\path\to\project\cumulusci.yml``
+
+This ``cumulusci.yml`` file lives in the project root directory and applies solely to this project. Changes here are committed back to a remote repository so other team members can benefit from the customizations. Configurations in this file apply solely to this project, and take precedence over any configurations specified in the global ``cumulusci.yml`` file, but are overridden by configurations in the local project ``cumulusci.yml`` file.
+
+
+Local Project Configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**macOS/Linux:** ``~/.cumulusci/project_name/cumulusci.yml``
+
+**Windows:** ``%homepath%\.cumulusci\project_name\cumulusci.yml``
+
+Configurations in this ``cumulusci.yml`` file apply solely to the project with the given <project_name>, and take precedence over *all other* configuration scopes except the universal ``cumulusci.yml`` file.
```

Comment:
This statement doesn't make sense. Doesn't the local project configuration override all scopes?

Reply:
No it doesn't, I've tested this multiple times. If I set the `util_sleep`'s `--seconds` option to 3 in local project cumulusci.yml and 10 in my project's cumulusci.yml, when I run the task it sleeps for three seconds. If I remove the configuration on the local project scope, then it will sleep for 10 seconds.

Reply:
Actually, I believe this makes sense. This way when you want to configure something over-top of what's in your project's configuration, then yours will take override the project. I made [this diagram](https://lucid.app/lucidchart/invitations/accept/bed65497-0a4e-4e8d-9cb1-bac38352d826) outlining the scopes months back. I think we aren't using it because we determined it would just be another image to maintain over time.

Reply:
That's right, this file is for personal, project-specific overrides.

Reply:
The "except the universal ``cumulusci.yml`` file" part doesn't make sense though.

Reply:
The "universal `cumulusci.yml` file" is just how we refer to the cumulusci.yml file that ships with CumulusCI. It originated from how the code base refers to this as the `universal_config`. I'm definitely in favor of changing this to something else. We refer to the "standard library" in the new doc set. @davisagli @davidmreed - would you be okay with referring to this file as the "Standard Library" cumulusci.yml file?

Reply:
I know what the universal config is and don't mind using that term in the docs. The problem here is that it says the universal config takes precedence over Local Project Configurations, which is just wrong.

Reply:
I should have been more specific here, I was pointing to the "except" clause as not making sense. I'm fine with the "universal" terminology, but based on @Br4nd0R's comment above, the "except" clause is incorrect. Or am I completely misreading?

Reply:
apologies. Yes, the except clause is incorrect. I'll remove it.